src/libtrx/game/interpolation.c

```diff
@@ -184,7 +184,9 @@ void Interpolation_Commit(void)
         }
 
         g_Camera.interp.room_num = g_Camera.pos.room_num;
-        Camera_ClampInterpResult();
+        if (g_Camera.type != CAM_PHOTO_MODE) {
+            Camera_ClampInterpResult();
+        }
```

Comment:
I think we'll need this too to set the room number effectively as currently there are some black screen flickers when moving around (for me at least, if you move around Lost Valley main area going up and down between rooms).
```
else {
    Room_GetSector(
        g_Camera.interp.result.pos.x,
        g_Camera.interp.result.pos.y + g_Camera.interp.result.shift,
        g_Camera.interp.result.pos.z, &g_Camera.interp.room_num);
}
```